<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1315</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1312</width>
    <height>0</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="load_button">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>110</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Görsel Yükle</string>
   </property>
  </widget>
  <widget class="QLabel" name="original_label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>90</y>
     <width>550</width>
     <height>410</height>
    </rect>
   </property>
   <property name="text">
    <string>Orijinal Görsel</string>
   </property>
  </widget>
  <widget class="QLabel" name="processed_label">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>90</y>
     <width>550</width>
     <height>410</height>
    </rect>
   </property>
   <property name="text">
    <string>İşlenmiş Görsel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="edge_button">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>660</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Kenar Tespit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sharpen_button">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>660</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Keskinleştir</string>
   </property>
  </widget>
  <widget class="QSlider" name="brightness_slider">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>560</y>
     <width>240</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-100</number>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="contrast_slider">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>600</y>
     <width>240</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimum">
    <number>10</number>
   </property>
   <property name="maximum">
    <number>300</number>
   </property>
   <property name="value">
    <number>100</number>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>560</y>
     <width>62</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Parlaklık:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>600</y>
     <width>62</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Kontrast:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="hist_button">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>560</y>
     <width>180</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Histogram Göster</string>
   </property>
  </widget>
  <widget class="QPushButton" name="auto_enhance_button">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>600</y>
     <width>180</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Otomatik İyileştir</string>
   </property>
  </widget>
  <widget class="QPushButton" name="highlight_button">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>640</y>
     <width>180</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Anormal Alanları İşaretle</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="canny_thresh1_spin">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>560</y>
     <width>70</width>
     <height>30</height>
    </rect>
   </property>
   <property name="maximum">
    <number>500</number>
   </property>
   <property name="singleStep">
    <number>1</number>
   </property>
   <property name="value">
    <number>100</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="canny_thresh2_spin">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>600</y>
     <width>70</width>
     <height>30</height>
    </rect>
   </property>
   <property name="maximum">
    <number>500</number>
   </property>
   <property name="singleStep">
    <number>1</number>
   </property>
   <property name="value">
    <number>150</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="area_threshold_spin">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>670</y>
     <width>70</width>
     <height>30</height>
    </rect>
   </property>
   <property name="maximum">
    <number>10000</number>
   </property>
   <property name="singleStep">
    <number>1</number>
   </property>
   <property name="value">
    <number>500</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>520</y>
     <width>210</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Kenar algılamada hassasiyetin</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>560</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Min:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>600</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Max:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>640</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Çember Alanı:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="undo_button">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>30</y>
     <width>110</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Geri</string>
   </property>
  </widget>
  <widget class="QPushButton" name="redo_button">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>30</y>
     <width>110</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>İleri</string>
   </property>
  </widget>
  <widget class="QPushButton" name="reset_button">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>30</y>
     <width>110</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Sıfırla</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
